### DISCUSSION TITLE
Developer mode

### DISCUSSION THREAD

**Discussion by mbensoussan** [OpenMage/magento-lts#2552] [Questions & Answers] [answered]
Hi,
before installing Openmage, i uncomment in index.php the lane Mage::setIsDeveloperMode(true) to see the error when i have http error 500 message in a page.
with openmage .. i don't see this lane ... 
so how i can do ?

**Comment by Flyingmana** [comment]
you can still use the same line in the same place, we just removed lines which were commented by default

**Comment by kiatng** [comment, marked as answer]
`Mage::setIsDeveloperMode(true);` is now here: https://github.com/OpenMage/magento-lts/blob/926e005f4840df9b6507e0f178d3856d2990ad62/app/Mage.php#L27-L32

So now, we can enable/disable developer mode in server config, for example, we can disable in production server or enable it in staging server without modifying `index.php`.

**Comment by elidrissidev** [comment]
If you're using apache you can add `SetEnv MAGE_IS_DEVELOPER_MODE 1` inside your site's `<VirtualHost>` node (only on dev environment!).

**Reply by addison74** [reply]
... or by editing .htaccess in root directory. In the future we should add this line with a comment in front of.

**Reply by elidrissidev** [reply]
Yes, I forgot that .htaccess is also a correct place to put it.

**Comment by mbensoussan** [comment]
Thank You for all you answer different solution, all work ;)

**Comment by sreichel** [comment]
If using DDEV put this to `.ddev/config.yaml`

```
web_environment: [
    MAGE_IS_DEVELOPER_MODE=1
]
```

**Reply by addison74** [reply]
Please mention this tip in PR #2575.
